src/components/common/PeopleAndRoles/OrgPerson.vue

```diff
@@ -94,18 +93,19 @@
                 </article>
 
                 <label v-else class="sub-header">{{ orgTypesLabel }} Name</label>
-                <article class="org-name-container pt-6">
+
+                <article class="org-name-container mt-4">
                   <v-checkbox
                     v-if="!toggleBcLookUp"
-                    class="mb-6"
+                    class="mt-0 pt-0 mb-4"
```

Comment:
This is to remove the default (Vuetify) top whitespace, since we want this component to be tight up against this article's bounding box.